packages/commands/src/components/style.scss

```diff
@@ -36,6 +36,9 @@
 }
 
 .commands-command-menu__container {
+	// the style here is a hack to force safari to repaint to avoid a style glitch
```

Comment:
I know this is terribly nitpicky, but do we have a standard for code comments? I normally use sentence capitalization and end with a period 🙈 

It's not a strong opinion, we just have a mix of styles.